recipes/ai-rag-amx-ubuntu/README.md

```diff
@@ -0,0 +1,47 @@
+<p align="center">
+  <img src="https://github.com/intel/optimized-cloud-recipes/blob/main/images/logo-classicblue-800px.png?raw=true" alt="Intel Logo" width="250"/>
+</p>
+
+# Intel® Corporation Copyright
+
+**Contributors: William Fowler, Joshua Segovia, Chris Ah-Siong**
+
+# Intel® Gen AI RAG DEMO using Xeon AMX
+
+## Description 
+
+The purpose of this demo aims to demonstrate the powerful capabilities of retrieval augmented generation based on large language models deployed on an Intel Xeon AWS instance (M7i.8xlarge), where we highlight performance optimizations leveraging Intel instructions to deliver increased accuracy and speed of inference. 
```

Comment:
Could we remove all comments around AWS and the Instance  `Intel Xeon AWS instance (M7i.8xlarge).`
Use `4th or 5th Gen Intel Xeon instances with Intel AMX` instead?